src/main/kotlin/no/nav/familie/ef/sak/repository/domain/BehandlingHistorikk.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import no.nav.familie.ef.sak.api.dto.BehandlingsHistorikkDto
+import no.nav.familie.ef.sak.service.steg.StegType
+import org.springframework.data.annotation.Id
+import java.time.LocalDateTime
+import java.util.*
+
+data class BehandlingHistorikk(@Id
```

Comment:
Liten sak men navnene er ikke helt konsekvente.
`BehandlingHistorikk`
`BehandlingsHistorikk`

Teamet ble tidligere enige om att gå for navn som er `Behandlingshistorikk` som er det "norske navnet"